docs/buildah-add.md

```diff
@@ -0,0 +1,32 @@
+## buildah-add "March 2017"
+
+## NAME
+buildah add - Add the contents of a file, URL, or a directory to the container. 
+
+
+## SYNOPSIS
+**buildah** **add** **containerID** **SRC** [**DEST**] 
+
+## DESCRIPTION
+Adds the contents of a file, URL, or a directory to a container's working directory.  If a local file appears to be an archive, its contents are extracted and added instead of the archive file itself.  If a destination is not specified, the location of the source will be used for the destination.  
+
+## EXAMPLE
+**buildah add containerID '/myapp/app.conf' '/myapp/app.conf' **
+**buildah add containerID '/home/myuser/myproject.go' **
+**buildah add containerID '/home/myuser/myfiles.tar' '/tmp' **
+**buildah add containerID '/tmp/workingdir' '/tmp/workingdir' **
+**buildah add containerID 'https://github.com/projectatomic/buildah' '/tmp' **
+
+## SEE ALSO
+buildah-commit(1)
```

Comment:
Lets just add a buildah(1) man page  and only refer to the others there.  Perhaps refer to related man pages, but not all, otherwise we will always be updating them all.
buildah-rmi and buildah-images, buildah-from, buildah-commit are related.